WcaOnRails/config/locales/zh-CN.yml

```diff
@@ -1337,8 +1356,6 @@ zh-CN:
       avatar_guidelines:
         #original_hash: 255f28c
         '1': 头像必须是真人照片。
-        #original_hash: 9126b0c
-        '2': 上面的人必须是你自己。
```

Comment:
hints

Reply:
Btw where can I find this in \simple_form\hints?

Reply:
Yes

Reply:
The line numbers are shown on the left. In en.yml it should be under `avatar_guidelines`.